lib/src/editor/ui/styled_icon_button.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import 'package:flutterbook/src/utils/extensions.dart';
+
+class StyledIconButton extends StatelessWidget {
+  final IconData icon;
+  final VoidCallback onPressed;
+
+  StyledIconButton({
+    required this.icon,
+    required this.onPressed,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return TextButton(
```

Comment:
This seem off to use a TextButton here when we aren't trying to show any text. Would it be more clear to use `GestureDetector` or `Inkwell`?

Reply:
That sounds good to me

Reply:
Actually, look at `IconButton` I just ran across this widget looking for something else.